### ISSUE TITLE
Polygon onPress coordinate is not returned on iOS (with GoogleMaps)

### ISSUE THREAD

**Issue by Pardiez** [react-native-maps/react-native-maps#4012]
# Bug report

## Summary

After [Fix: polygon on press wrong coordinates (#3966)](https://github.com/react-native-maps/react-native-maps/pull/3966) by @Filippo39 I can't get the pressed coordinate on Android but not in iOS (using GoogleMaps on both). Of course, It wasn't working before that PR, but It seems still broken for iOS, at least with GoogleMaps.

## Environment info

      react-native: 0.63.3
      react: 16.13.1
      react-native-maps: "git://github.com/react-native-maps/react-native-maps.git#b2f8e765a799b2b0d2a77c49ff254fa4d4efc02f"
    Target Platform: eg: iOS - 11.0 Simulator  (using GoogleMaps)
      
Also tested with ([master branch 11/17/2021](https://github.com/react-native-maps/react-native-maps/commit/d3bf76336920098f5d0e539f5787e498a41420a4))

`react-native info` output:

```bash
System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 59.50 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 8.1.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /Users/david/.rvm/gems/ruby-2.6.3/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.2, DriverKit 20.0, macOS 11.0, tvOS 14.2, watchOS 7.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7199119
    Xcode: 12.2/12B45b - /usr/bin/xcodebuild
  Languages:
    Java: 13.0.1 - /usr/bin/javac
    Python: 3.9.0 - /Users/david/.pyenv/shims/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

## Steps to reproduce

Using GoogleMaps (I have no tested it with AppleMaps) and pressing any geometry in IOS event.nativeEvent has no coordinate `{"action": "polygon-press", "id": "unknown", "target": 4293}`
I expected the pressed coordinate to be there, like in Android: `{"action": "polygon-press", "coordinate": {"latitude": 41.92031979841978, "longitude": -4.776083864271641}, "position": {"x": 217, "y": 472}}`

## Reproducible sample code

```
<Polygon onPress={(event) => { console.log(event.nativeEvent) }}/>
```
I suppose no repro is needed. Please let me know if I'm wrong.

**Comment by christopherdro** [comment]
Yea it looks like the addition on iOS was only for Apple maps and not Google maps. @Filippo39 Any chance you can follow up with adding the same functionality for Google Maps on iOS?

**Comment by Pardiez** [comment]
I guess changes shouldn't be released if they don't behave the same way in both libraries. @Filippo39

**Comment by isidoro98** [comment]
> Fix: polygon on press wrong coordinates (#3966)

Hey @Pardiez I've opened a PR that should fix your issue #4036

**Comment by Pardiez** [comment]
Thank you @isidoro98 !. I will check it
